changelogs/changelog.yaml

```diff
@@ -78,6 +78,20 @@ releases:
       name: zos_tso_command
       namespace: ''
     release_date: '2022-06-07'
+  1.10.0:
+    changes:
+      release_summary: 'Release Date: ''2024-06-11''
```

Comment:
Shouldn't this be 2024-06-21?

Reply:
well it was created during the release process so is inaccurate